src/ResourceManagement/Automation/AutomationManagement/Generated/HybridRunbookWorkerGroupOperations.cs

```diff
@@ -413,11 +413,11 @@ public AutomationManagementClient Client
                                         hybridRunbookWorkerInstance.IpAddress = ipAddressInstance;
                                     }
                                     
-                                    JToken lastSeenValue = hybridRunbookWorkersValue["lastSeen"];
-                                    if (lastSeenValue != null && lastSeenValue.Type != JTokenType.Null)
+                                    JToken lastSeenDateTimeValue = hybridRunbookWorkersValue["lastSeenDateTime"];
+                                    if (lastSeenDateTimeValue != null && lastSeenDateTimeValue.Type != JTokenType.Null)
                                     {
-                                        DateTimeOffset lastSeenInstance = ((DateTimeOffset)lastSeenValue);
-                                        hybridRunbookWorkerInstance.LastSeenDateTime = lastSeenInstance;
+                                        DateTimeOffset lastSeenDateTimeInstance = ((DateTimeOffset)lastSeenDateTimeValue);
```

Comment:
It looks like there are some commits which which should be in the master are not reflecting in it. Could you do a rebase and push the branch?